sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/test/ut/blob_checkpoint_store_test.cpp

```diff
@@ -7,6 +7,7 @@
 #include <azure/core/context.hpp>
 #include <azure/identity.hpp>
 #include <azure/messaging/eventhubs.hpp>
+#include <azure/storage/blobs/blob_container_client.hpp>
```

Comment:
If you need to explicitly include this header after including eventhubs headers, then the blob checkpointstore's headers should include all the necessary Storage headers. I know, it is possible to forwerd-declare almost everything, but our SDK does not follow this path - the experience everywhere else is that you include the file for the class you need, and everything you can pass to its constructors is also included.
And it doesn't look like you should include this header - `blob_checkpoint_store.hpp` does include `azure/storage/blobs.hpp`, and on that - I recommend to explicitly include only the headers that `BlobCheckpointStore` really uses, so in `blob_checkpoint_store.hpp` to replace `azure/storage/blobs.hpp` with specific storage header(s).

Reply:
Ah, you're right - this header is unnecessary.

So are a bunch of blob headers, cleaned up.